go.mod

```diff
@@ -0,0 +1,19 @@
+module github.com/ONSdigital/dp-table-renderer
+
+go 1.13
+
+require (
+	github.com/360EntSecGroup-Skylar/excelize v1.2.1-0.20180108150359-50cdaed5a36f
+	github.com/ONSdigital/dp-healthcheck v1.0.5
+	github.com/ONSdigital/dp-net v1.0.5-0.20200805150805-cac050646ab5
```

Comment:
can we update to a tagged version, please?

Reply:
Should be updated to `1.0.9`